README.md

```diff
@@ -1 +1,88 @@
-# Books-store
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Book Store
+
+> This is an Educational project aimed at learning professional use of React/React-Redux. It features books management; Adding, Editing, Completing and Removing books from the system using Redux actions and reducers. This app is MIT Licensed
+
+## Built With
+
+- JavaScript
+- ReactJs,
+- Git, GitHub, Git workflow
+
+## Live Demo
+
+[Link]()
+
+## Getting Started
+
+### To have a copy of this project running on your local machine, follow these commands:
+
+- Clone this repo to your local machine
+  - `https://github.com/AshwinCarvalho999/Books-store.git`
+  - Open the folder "Books-store and run `npm install` to install all necesary scripts,
+  - Run from your termminal, `npm start` to open the project in your browser.
+
+**This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).**
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- [NodeJS](https://nodejs.org/en/docs/)
+
+> As an asynchronous event-driven JavaScript runtime, Node.js is designed to build scalable network applications. In the following "hello world" example, many connections can be handled concurrently. Upon each connection, the callback is fired, but if there is no work to be done, Node.js will sleep.
+
+- [Text Editor(VScode Recommended)](https://code.visualstudio.com/)
+
+> Visual Studio Code is a free coding editor that helps you start coding quickly. Use it to code in any programming language, without switching editors.
+
+### Setup
+
+**Run from your terminal:**
+
+- `npx create-react-app todo-list-react`
+
+**cd into Books-store folder, then run from your terminal:**
+
+- `npm start`
```

Comment:
- [ ] Kindly add `npm install` command between these two commands because you will need to install packages before you start the application 👍